docs/07-AdvancedUsage.md

```diff
@@ -267,20 +269,12 @@ Alternatively, the `@dataprovider` can also be a public method starting with `_`
     */
     public function staticPages(AcceptanceTester $I, \Codeception\Example $example)
     {
-        $I->amOnPage($example['url']);
-        $I->see($example['title'], 'h1');
-        $I->seeInTitle($example['title']);
+        // ...
```

Comment:
I deleted the parts which are the same as in the above code block. Here, only the syntax of the public data provider is important, the rest is obvious :-)